gulpfile.js

```diff
@@ -0,0 +1,50 @@
+var gulp = require('gulp');
```

Comment:
I'm about 95% sure that this file isn't needed anymore. It's leftover from when I was using a separate **node** job in Travis to build the Swagger UI.

Reply:
I'm not certain if you mean the whole js file is not needed or just this include.

I'm adding this as a task for https://github.com/ga4gh/data-repository-service-schemas/issues/203